wp-content/themes/access/views/programs/single.twig

```diff
@@ -127,6 +135,24 @@
       </div>
     </section>
 
+    <div itemprop="governmentBenefitsInfo" itemtype="http://schema.org/GovernmentService" itemscope>
```

Comment:
This block should be conditionally present. Only when the item scope is "specialannouncement" or the program status is "covid-response"